docs/testplan/BGP-BGPSentinel.md

```diff
@@ -0,0 +1,51 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+## Overview
+The purpose is to test that BGPSentinel (BGPS) host will setup an IBGP connection to FRR. BGPSentinel (BGPS) will advertise and withdraw routes to FRR. As BGPSentinel session is IBGP and not directly connected to the DUT, this feature relies on V4 and V6 nht resolve via default configuration in FRR.
+
+### Scope
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is not to test specific API, but functional testing of BGPSentinel configuration on SONIC system.
+
+### Testbed
+The test will run on the following testbeds:
+* t1-lag (vs)
+
+## Setup configuration
+This test doesn't require any configuration in ansible deployment. IBGP session between BGPSentinel (BGPS) host and DUT are created in set up phase of this case and are cleaned in tear down phase of this case. BGPSentinel (BGPS) host is simulated by exabgp in PTF.
+
+## Test
+The test will configure BGPSentinel session and then check that IBGP session would be setup between BGPSentinel (BGPS) host and DUT. After session set up, BGPSentinel (BGPS) will advertise routes with higher local-preference and no-export community. These routes will act as best-path in DUT, which would suppress routes received from t0. The test will cover both V4 and V6 BGPSentinel session.
+
+## Test cases
```

Comment:
Based on what we discussed during community Testplan review, can you please include the following:
- have a test with bgp soft & hard reset triggers and verify that the DUT is still honoring the right best path from the BGPSentinel peer and suppressing it accordingly to T2 peers

Reply:
Add Test case  3 - Soft/Hard reset IBGP V4 and V6 Session in DUT to cover this part. Thanks.